README.md

```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
 
 $ `git clone https://github.com/ita-social-projects/ZeroWaste.git`
 
-## Setup
+## Local setup
+First of all you need rvm to setup project. For the operating system Windows the optimal solution is to use <a href="https://docs.microsoft.com/en-us/windows/wsl/">WSL</a>.
 
 $ `bin/setup`
-or 
-$ `bundle install` 
+or
+$ `bundle install`
+
+<b>PostgreSQL</b>
+
+<a href="https://www.postgresql.org/download/">Install PostgreSQL</a> for your operating system or subsystem.
+You can familiarize yourself with <a href="https://www.postgresql.org/docs/">PostgreSQL documentation</a>.
+
+In your local machine in cloned project in config folder rename database.yml.sample to database.yml. Make sure that the user and password match the data in this file. Port may be changed.
+
+<b>pg gem</b>
+
+Under certain circumstances bundle can do not install pg.
+
+To install manually:
+$ `sudo apt-get install libpq-dev`
+then
+$ `gem install pg`
+
```

Comment:
There were a lot of issues with Postgres, which are worth mentioning. Two of the most popular ones:
https://stackoverflow.com/questions/15301826/psql-fatal-role-postgres-does-not-exist
https://stackoverflow.com/questions/55038942/fatal-password-authentication-failed-for-user-postgres-postgresql-11-with-pg